csharp/src/Ice/OutgoingRequest.cs

```diff
@@ -161,15 +161,23 @@ public sealed class OutgoingRequestWithStructParam<TParamList, TReturnValue>
     // publicly visible Ice-internal class used as base class for void requests
     public class OutgoingRequest
     {
+        private readonly bool _oneway;
+
+        private protected OutgoingRequest(bool oneway)
```

Comment:
That's fine, but you could also use the syntax:
```
private protected OutgoingRequest(bool oneway) => _oneway = oneway;
```

Reply:
You're totally right! I guess I haven't let the expression bodies sink in enough yet!